### PR TITLE
Implement Landing Page Basics

### PR DESCRIPTION
**Pull request by Bloo-dev**
This implements some fundamental ideas for the landing page. It is by far not perfect, nor is it 100% complete but it should provide a good starting point.

The PR is still a draft as we will do some double-checking tomorrow.

Attached are two demo views of the landing page, one for widescreen devices and one for portrait mode.
widescreen
potrait

### CODE REVIEW

File: site/includes/homepage.css [hunk 1]
```diff
@@ -23,7 +25,7 @@ CSS for the module browse page
 }
 
 body{
-  font-family: Verdana,Geneva,sans-serif;
+  font-family: 'Raleway',sans-serif;
```
Comment (sparks0) [inline site/includes/homepage.css:28]: The reason the main body doesn't use Raleway as its font is that numbers do not display well in this font. If we go this route we will need to add some sort of number span to display digits.

File: site/includes/homepage.css [hunk 2]
```diff
@@ -73,23 +88,50 @@ a:visited{
...
+        display: none;
+    }
+}
+
+#header ul {
   position: relative;
   top: 0px;
   left: 0px;
-  min-height: 41px;
+  min-height: auto;
```
Comment (sparks0) [inline site/includes/homepage.css:107]: This should probably just be height:auto. Before your change the intent of this code was to give the header a height of 41px, but allow it to get deeper if, say, the page was too narrow and some of the buttons wanted to newline. Using min-height would still allow it to expand downwards to accomodate height changes.

If you are handling height changes programatically now, this should probably be height:auto not min-height

File: site/indexNew.php [hunk 3]
```diff
@@ -44,25 +44,42 @@
...
-  </ul>
-</div>
-<div id="slideshowContainer">
-  <div id="slideshowLeft" class="slideshowArrow" onclick="slideshow('left')"></div>
-  <div id="slideshowRight" class="slideshowArrow" onclick="slideshow()"></div>
-  <div id="slideshowSubContainer" class="cardContainer">
-    
+<div id="landingGroup" width="100%">
+  <div id="headerGroup">
+    <div id="header" style="display: flex; margin-top: 4px">
```
Comment (sparks0) [inline site/indexNew.php:49]: I'm guessing they're here cos you're still testing and this is a draft but these styles should be moved to the stylesheet.

File: site/includes/homepage.css (hunk 1 above)
Comment (jojomatik) [reply site/includes/homepage.css:28]: Adding `font-variant-numeric: lining-nums;` "correctly" aligns the numbers with the rest of the text. The mixed fonts just looked a bit off to me.

**Comment by Bloo-dev** [comment]
This is now ready to be merged. Here are some updated screenshots (not much changed, apart from the inclusion of the theme button in the header and a slight color change to horizontal lines as well as proper positioning of numbers):
widescreen
portrait

**Pull request merged**